Automattic/wp-calypso
# Anchor: Update branding in the My Home podcasting card.

Anchor recently updated their branding. This PR updates their logo in the podcasting My Home card to be compliant with their new usage guidelines.

<img width="1231" alt="Screen Shot 2021-02-04 at 2 41 45 PM" src="https://user-images.githubusercontent.com/349751/106964579-2b0a9080-66f7-11eb-8b96-eb1c0954f510.png">

Fixes 441-gh-Automattic/dotcom-manage

**Testing Instructions**
* Load this branch in a local dev environment, or use the calypso.live branch with `?flags=anchor-fm-dev`.
* One a site with at least three published posts, cycle through and dismiss all of the My Home cards until you get to the podcasting card.
* Verify the Anchor logo is the updated purple one shown above.
